authorization_django/middleware.py

```diff
@@ -166,11 +166,11 @@ def get_claims(jwt):
                 'scopes': {convert_scope(r) for r in claims['realm_access']['roles']},
                 'claims': claims
             }
-        elif claims.get('scp') and claims.get('preferred_username'):
-            # Microsoft token structure
+        elif claims.get('groups') and (claims.get('unique_name') or claims.get('un')):
```

Comment:
Why not support both? Or was the previous already designed for Azure?

Reply:
Previous was designed for azure indeed, but never used (surprise!) and apparently azure is not using it anymore :/